refinery/tool_manager/models.py

```diff
@@ -557,8 +557,8 @@ def _get_detailed_nodes_dict(self, node_uuid_list,
                 ),
                 self.AUXILIARY_FILE_LIST: [
                     get_file_url_from_node_uuid(
-                        child.uuid, require_valid_url=require_valid_urls
-                    ) for child in Node.objects.get(uuid=node["uuid"]).
+                        uuid, require_valid_url=require_valid_urls
+                    ) for uuid in Node.objects.get(uuid=node["uuid"]).
```

Comment:
Why not just: `get_file_url_from_node_uuid(node['uuid'], require_valid_url=require_valid_urls)`?

Reply:
this is actually what tripped me up and caused the error - not sure how to make it more readable but we are actually getting a list of `uuid`'s from `get_auxiliary_nodes` on the next line

Reply:
Oh, I missed this too. This whole dict comprehension (`node_info`) is way too large. I think it would be much cleaner to rewrite as a simple for loop to avoid problems like this.